src/main/java/io/penguinstats/controller/ResultController.java

```diff
@@ -170,152 +100,4 @@
 		}
 	}
 
-	@ApiOperation("Get segmented drop data for all items in all stages")
-	@GetMapping(path = "/trends", produces = "application/json;charset=UTF-8")
-	public ResponseEntity<String> getAllSegmentedDropResults(
```

Comment:
This method is not marked with TODO, but still deprecated?

Reply:
In the legacy result controller only the first method is still being used, so I delete this one.